docs/docs-site/content/administrator/configuration/connectors/_index.md

```diff
@@ -803,6 +803,21 @@ Alternative:
     ## The JDBC driver clickhouse-jdbc.jar and its related jars need to be in the CLASSPATH environment variable.
     options='{"url": "jdbc:clickhouse://localhost:8123", "driver": "ru.yandex.clickhouse.ClickHouseDriver", "user": "readonly", "password": ""}'
 
+
+### Apache Kyuubi
+
+[Apache Kyuubi](https://kyuubi.readthedocs.io/en/master/) is a distributed and multi-tenant gateway to provide serverless SQL on data warehouses and lakehouses.
```

Comment:
would you mind replacing the link with the project the Apache site instead of the docs site?
```suggestion
[Apache Kyuubi](https://kyuubi.apache.org/) is a distributed and multi-tenant gateway to provide serverless SQL on data warehouses and lakehouses.
```